examples/grafana/README.md

```diff
@@ -109,4 +109,27 @@ A second example dashboard `fn_grafana_dashboard2.json` in this example's direct
 In the following screenshot, the "Choose spans to display rates" dropdown has been used to select `agent_submit` and `serve_http`, and the "Choose spans to display durations" dropdown, has been used to select `agent_cold_exec`, `agent_get_slot`, `agent_submit`, `docker_create_container`, `docker_start_container` and `docker_wait_container`. 
 
 <img src="../../docs/assets/GrafanaDashboard2.png" width="100%">
+
+## Docker statistics
+
+During the execution of the docker container, a selected number of statistics from docker are available as Prometheus metrics. The available metrics are listed in the following table:
+
+| Prometheus metric name |
+| ------------- |
+| `fn_docker_stats_fn_cpu_kernel` |
+| `fn_docker_stats_fn_cpu_kernel`  |
+| `fn_docker_stats_fn_cpu_user` |
+| `fn_docker_stats_fn_disk_read` |
+| `fn_docker_stats_fn_disk_write` |
+| `fn_docker_stats_fn_mem_limit` |
+| `fn_docker_stats_fn_mem_usage` |
+| `fn_docker_stats_fn_net_rx` |
+| `fn_docker_stats_fn_net_tx` |
```

Comment:
these seem to have redundant `fn_` bits, perhaps we could remove the latter? `docker_stats_net_rx` seems pretty well scoped, regardless of the `fn_` prefix.

Reply:
The second half of these metric names (e.g. `fn_mem_limit`) is the name of the key by which the value in question is logged to the span. The Prometheus collector uses this to decide which of the values logged with the span to convert to Prometheus metrics (we don't want it turning every random logged value into a metric).

However it's easy to make the Prometheus collector strip this off when it constructs the metric name. I'll do this.

Reply:
seems reasonable. 

> we don't want it turning every random logged value into a metric

actually, i think this is okay / desirable. it's just us logging metric events I think, would be nice to have. we have stderr logging for all else.